tests/ManageCourses.Tests/UnitTesting/Helpers/TestHelper.cs

```diff
@@ -1,314 +1,305 @@
 using System.Linq;
 using GovUk.Education.ManageCourses.Domain.DatabaseAccess;
+using GovUk.Education.ManageCourses.Domain.EqualityComparers;
 using GovUk.Education.ManageCourses.Domain.Models;
 using Microsoft.EntityFrameworkCore;
 
 namespace GovUk.Education.ManageCourses.Tests.UnitTesting.Helpers
 {
-    internal static class TestHelper
+    internal class TestHelper
     {
-        private static ManageCoursesDbContext _dbContext;
+        public ManageCoursesDbContext DbContext { get; private set; }
 
-        public static string OrgWithProviderEmail { get; } = "someuser@somewhere.com";
+        public const string EmailWithProvider = "someuser@somewhere.com";
+        public const string EmailNoProvider = "someotheruser@somewhereelse.com";
+        public const string EmailMultiOrg = "userwithmultipleorgs@somewhere.com";
+        public const string EmailMultiInst = "userwithmultipleucascodess@somewhere.com";
+        public const string UcasInstCodeWithProviders = "134";
+        public const string UcasInstCodeNoProviders = "B20";
+        private const string StMichaelSCatholicCollege = "St Michael's Catholic College";
+        private const string MultiOrgInstCode1 = "2GG";
+        private const string MultiOrgInstCode2 = "2G8";
+        private const string MultiOrgInstCode3 = "N43";
 
-        public static string OrgWithNoProviderEmail { get; } = "someotheruser@somewhereelse.com";
-
-        public static string UserWithMultipleOrganisationsEmail { get; } = "userwithmultipleorgs@somewhere.com";
-        public static string UserWithMultipleOrganisationUcasCodesEmail { get; } = "userwithmultipleucascodess@somewhere.com";
-
-        public static string OrgUcasCodeWithProviders { get; } = "134";
-
-        public static string OrgUcasCodeNoProviders { get; } = "B20";
+        public TestHelper()
+        {
+            SetupInMemoryContext();
+        }
 
-        public static ManageCoursesDbContext GetFakeData(EnumTestType testType)
+        public TestHelper(ManageCoursesDbContext dbContext)
         {
-            SetupContext();
-            switch (testType)
-            {
-                case EnumTestType.DataService:
-                    BuildFakeDataForService();
-                    break;
-                case EnumTestType.DataHelper:
-                    BuildFakeDataForHelper();
-                    break;
-            }
-            return _dbContext;
+            DbContext = dbContext;
         }
 
-        private static void SetupContext()
+        private void SetupInMemoryContext()
         {
             var optionsBuilder = new DbContextOptionsBuilder<ManageCoursesDbContext>();
             optionsBuilder.UseInMemoryDatabase("dbInMemory");
-            _dbContext = new ManageCoursesDbContext(optionsBuilder.Options);
+            DbContext = new ManageCoursesDbContext(optionsBuilder.Options);
         }
 
-        private static void BuildFakeDataForHelper()
+        public void BuildFakeDataForHelper()
```

Comment:
Static and pass in context?

Reply:
lots of things in this class depend on hitting the context field, I think including methods this calls.